### ISSUE TITLE
Feedback from me about node.js, ruffle and vscode

### ISSUE THREAD

**Issue by A-few-degrees-elegance** [i486/VxKex#25]
The v18.20.4 version of Node.js can be used normally. I can even use Vite to create a Vue3 project and successfully run the server (it's worth mentioning that this version of Node is best downloaded using RAR instead of MSI)

The latest version of VSCode (v1.19.2) can be opened normally, but cannot open terminals and shortcuts. I can only refund v1.70.2 for use.

Ruffle's exe version cannot be used. Okay, this is a niche software

In summary, Node.js has been a great help to me, and I hope you can continue to update it.
Love comes from Hangzhou, China

**Comment by A-few-degrees-elegance** [comment]
![66406b1562ff1b1e9f5baa3fd56a189](https://github.com/user-attachments/assets/88287f23-449b-4634-ae67-0966b56b21ec)

**Comment by A-few-degrees-elegance** [comment]
![Uploading 1729506839738.jpg…]()

**Comment by A-few-degrees-elegance** [comment]
![Uploading 1729506839738.jpg…]()

**Comment by A-few-degrees-elegance** [comment]
![Uploading fd5c8197c0f4ed80f5f5bb47db75b5c.png…]()

**Comment by A-few-degrees-elegance** [comment]
![Uploading fd5c8197c0f4ed80f5f5bb47db75b5c.png…]()

**Comment by A-few-degrees-elegance** [comment]
![Uploading fd5c8197c0f4ed80f5f5bb47db75b5c.png…]()

**Comment by A-few-degrees-elegance** [comment]
![Uploading fd5c8197c0f4ed80f5f5bb47db75b5c.png…]()

**Comment by A-few-degrees-elegance** [comment]
I want to send more pictures, but unfortunately my internet is too bad

**Comment by i486** [comment]
Thanks for the feedback.

We are closing this issue as part of a cleanup of long-standing inactive reports.

If this issue is still relevant and reproducible with the latest version, feel free to reopen or create a new issue with updated details.
